### PR TITLE
[EMAILS] Manage & test templates in the application

### PR DESCRIPTION
**Pull request by Remg**

### CODE REVIEW

File: templates/email/template/adherent_account_activation_message.html.twig [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+                  <p>
+                      Bonjour {{var:target_firstname:""}},
+                  </p>
+                  <p>
+                      Merci d'avoir rejoint le mouvement !
+                  </p>
+                  <p>
+                      Il ne vous reste plus qu’une étape : confirmer  votre adresse e-mail.
+                  </p>
+                </div>
```
Comment (atamano) [inline templates/email/template/adherent_account_activation_message.html.twig]: this ending div without opening div breaks email structure in gmail.
screen shot 2017-10-16 at 17 28 33

Comment (Remg) [reply templates/email/template/adherent_account_activation_message.html.twig]: @marineb could you check?

File: app/config/config.yml [hunk 2]
```diff
@@ -187,6 +187,13 @@ sonata_admin:
         layout:  admin/layout.html.twig
     security:
         handler: sonata.admin.security.handler.role
+    dashboard:
+        groups:
+            Emails:
+                items:
+                    - app.admin.email
+                    - route:        app_admin_email_template_list
```
Comment (HeahDude) [inline app/config/config.yml]: The alignement looks weird.

File: app/config/config.yml [hunk 3]
```diff
@@ -224,12 +231,12 @@ old_sound_rabbit_mq:
         mailjet_campaign:
             connection:       default
             exchange_options: { name: 'mailjet-delayed-campaign', type: direct }
-            class:            AppBundle\Producer\MailjetProducer
+            class:            AppBundle\Producer\MailerProducer
```
Comment (HeahDude) [inline app/config/config.yml]: Can you please base this PR on the other branch to have a lighter diff? I'm stopping here the review for now.

File: app/config/config.yml (hunk 2 above)
Comment (Remg) [reply app/config/config.yml]: Done

File: app/config/config.yml (hunk 3 above)
Comment (Remg) [reply app/config/config.yml]: Done

**Comment by HeahDude** [comment]
Please don't forget to target `develop`.

File: templates/email/layout.html.twig [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+    <table border="0" cellpadding="0" cellspacing="0" height="100%" width="100%" id="bodyTable">
+        <tr>
+            <td align="center" valign="top">
+                <table border="0" cellpadding="20" cellspacing="0" width="600" id="emailContainer">
+                    <tr class="email__preheader">
+                        <td></td>
+                    </tr>
+                    <tr class="email__header">
+                        <td align="center" valign="top">
+                            <a href="{{ url('homepage') }}" target="_blank">
```
Comment (Remg) [inline templates/email/layout.html.twig]: unclosed tag

**Comment by B-Galati** [comment]
Closing as duplicate of #2229

**Pull request closed without merging**